src/USER-GWBC/in.flow.pois.3D

```diff
@@ -0,0 +1,44 @@
+units		metal
```

Comment:
no example inputs in the source tree, please. those should be moved to the examples tree.